tests/helpers.bash

```diff
@@ -48,9 +48,17 @@ EOF
 function starthttpd() {
     pushd ${2:-${TESTDIR}} > /dev/null
     go build -o serve ${TESTSDIR}/serve/serve.go
-    HTTP_SERVER_PORT=$((RANDOM+32768))
-    ./serve ${HTTP_SERVER_PORT} ${1:-${BATS_TMPDIR}} &
+    ./serve ${1:-${BATS_TMPDIR}} 0 port &
```

Comment:
Pretty please add `rm -f port` . Even though I'm familiar with this code, I *still* had to convince myself that `$TESTDIR` is the one-off temporary working directory and not the shared common source one (the one with the `S`. I can never remember which one is which). An `rm` is a NOP, but it's a friendly-to-the-reader one.

Reply:
Added.

Reply:
I was suggesting adding it here, early, before referencing it (i.e. so the reader can be assured that there is no preëxisting "port" file). This is too minor to require a re-push though.